dora/core/server/master/src/main/java/alluxio/master/scheduler/DefaultWorkerProvider.java

```diff
@@ -54,6 +54,11 @@ public List<WorkerInfo> getWorkerInfos() {
     }
   }
 
+  @Override
+  public List<WorkerInfo> getLiveWorkerInfos() {
```

Comment:
just wanted to make sure, this does not throw any declared exception, meaning all errors will be RuntimeException?

Reply:
there should be, this needs to be further revamped when developing etcd interaction for scheduler